src/shared/parse.cpp

```diff
@@ -1723,8 +1713,8 @@ static void Parse_FreeDefine( define_t *define )
 	}
 
 	//free the define
-	free( define->name );
-	Z_Free( define );
+	BG_Free( define->name );
```

Comment:
This string is allocated with strdup, so I'd say for consistency it'd be better to keep `free()`. (For now this would work fine, but this relies on the current impl of `BG_Free`)

Reply:
That's a weird definition of consistency, considering BG_Free() is just free() anyway